### DISCUSSION TITLE
Trouble Pushing Files

### DISCUSSION THREAD

**Discussion by chazwilson01** [community/community#81372] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I keep getting this error when trying to push:

Uploading LFS objects:   0% (0/1), 0 B | 0 B/s, done.
batch response: @chazwilson01 can not upload new objects to public fork chazwilson01/chazwilson01.github.io
error: failed to push some refs to 'github.com:chazwilson01/chazwilson01.github.io.git'

if someone knows how to fix it, please help me.

**Comment by faridnec** [comment]
Hey, If you haven't already, run `git lfs install` to ensure LFS is properly set up and authenticate with LFS using `git lfs authenticate`. After that, check fork permission. Assuming you own the original repository, adjust permissions in the repository settings then retry the push using `git push`

**Comment by 4vinn** [comment]
Hello @chazwilson01 !

1. Firstly verify if LFS is enabled in the repo. (setting > options > Git LFS)
1. Make sure that Git LFS is installed and configured correctly on your local machine, run the following code:
  ```
git lfs install
git lfs track
```

And please go through [this](https://docs.github.com/en/repositories/working-with-files/managing-large-files) once.
